src/sentry/utils/distutils/commands/base.py

```diff
@@ -9,6 +9,14 @@
 from subprocess import check_output
 from distutils.core import Command
 
+import sentry  # We just need its path via __file__
```

Comment:
Are there any side effects of importing `sentry` into the base distutils module like this?

Reply:
Not as far as I know. Matt suggested this showing a similar usage in another place.